apps/ledger-live-desktop/src/renderer/modals/AddAccounts/steps/StepChooseCurrency.tsx

```diff
@@ -140,11 +140,13 @@ const StepChooseCurrency = ({ currency, setCurrency }: StepProps) => {
     const currencies = (listSupportedCurrencies() as CryptoOrTokenCurrency[]).concat(
       listSupportedTokens(),
     );
-    const deactivatedCurrencies = Object.entries(featureFlaggedCurrencies)
-      .filter(([, feature]) => !feature?.enabled)
-      .map(([name]) => name);
+    const deactivatedCurrencies = mock
```

Comment:
I modified this piece of code because sometimes, we need to launch playwrights tests against some featureFlag disable currencies.
So, I don't want to disable currencies with MOCK=1

Reply:
Still not sure if I understand, here you are setting all currencies to activated if it's `MOCK`? What is happening if you aren't doing this?

Reply:
For example, I've added optimism mock in this PR.
but the "optimism" is not available by default because of the deactivated featureflag.
If I don't have this change, the playwright test will fail  because of an "optimism is not supported" error. (the "deactivatedCurrencies" array contains "optimism")